### PR TITLE
fix(DB/creature_template): Mass update vanilla creatures damages

### PR DESCRIPTION
**Pull request by T1ti**
## Changes Proposed:
Based on the bestiary book : 
-Updated Molten Core, BWL and AQ 40 bosses damages
-Updated all Molten Core trash damages
**They all had a placeholder value, 35 for bosses and 7.5 for trash. Boss damages will be a lot lower overall but trash damages will be a lot higher**
-Updated a bunch of vanilla creatures by ID (checked creatures that had a modifier higher than 1, from ID 1 to 4625)
-updated a bunch of important creatures(msotly dungeon bosses) that used the placeholder value 7.5 (1050 creature still use it)

## Tests Performed:
-Querries runs properly
-checked some beasts in game with beast lore

## How to Test the Changes:
 - teleport to a creature modified by this PR with `.go c id xxx`
-let the creature hit you without armor to test the damages(nude character still has a bit of armor) or use beast lore and check the damages
-or just eyeball the changes/calculate it

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by T1ti** [comment]
I'm gonna stop there, it's already a lot.
Would be nice if we had a way to track the verified mobs.

**Comment by FrancescoBorzi** [comment]
thanks @T1ti , summoning @Knindzagxg

**Comment by Zoidwaffle** [comment]
Hm, could it be tracked in the VerifiedBuild column or perhaps add another column to creature_template?

**Comment by Knindzagxg** [comment]
Will check once im back tomorrow from work.

**Comment by T1ti** [comment]
Any testers ? :(

**Comment by Knindzagxg** [comment]
Had no time sadly, but will check during weekend mate.

**Comment by FrancescoBorzi** [comment]
@azerothcore/testers @pklloveyou please check this

**Comment by Knindzagxg** [comment]
Tested more then 60% and looks good so far.
Will keep checking a bit more later and tomorrow.

PS. We will have to check all TBC / Wotlk ones as well.

**Comment by T1ti** [comment]
> Tested more then 60% and looks good so far.
> Will keep checking a bit more later and tomorrow.
> 
> PS. We will have to check all TBC / Wotlk ones as well.

We don't have the book for Wotlk creatures.
Your best bet to fix wotlk creature damages is to just wait for Wotlk classic in 2 years.

File: data/sql/updates/pending_db_world/rev_1616275490460233200.sql [hunk 1]
```diff
@@ -263,3 +263,11 @@ UPDATE `creature_template` SET `DamageModifier`='6.5' WHERE  `entry`=11441;
...
 -- gordok mauler 5->1 (outside)
 UPDATE `creature_template` SET `DamageModifier`='1' WHERE  `entry`=11442;
+
+-- some deadmines bosses
+-- ogre
+UPDATE `creature_template` SET `DamageModifier`='2.5' WHERE  `entry`=644;
+-- greenskin
+UPDATE `creature_template` SET `DamageModifier`='2.45' WHERE  `entry`=647;
+-- cookie
+UPDATE `creature_template` SET `DamageModifier`='2.45' WHERE  `entry`=645;
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1616275490460233200.sql:275]: you can merge these two UPDATE query into one
```SQL
UPDATE `creature_template` SET `DamageModifier`='2.45' WHERE  `entry` IN (647, 645);

```

**Comment by Hacki95** [comment]
@Knindzagxg Did you have time to test these? I tested a few and they seem to reflect the values from the book. I would approve this PR.

**Comment by Knindzagxg** [comment]
I totally forgot to comment, even tho we talked on Disc xD
Didn't find any issues trough Raid or etc while testing, so yeah, everything is fine.

Yet again, sorry cause super late post, just super busy with work atm and 10hr shifts xD

**Review by Knindzagxg — APPROVED** [review]

**Comment by Hacki95** [comment]
All good! Thank you!

**Comment by T1ti** [comment]
is this getting merged ?

File: data/sql/updates/pending_db_world/rev_1616275490460233200.sql [hunk 2]
```diff
@@ -0,0 +1,285 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1616275490460233200');
+
+-- raid bosses
+-- Molten Core
+-- Lucifron 35 -> 16
+UPDATE `creature_template` SET `DamageModifier`='16' WHERE  `entry`=12118;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1616275490460233200.sql:6]: https://www.azerothcore.org/wiki/sql-standards#general-standards

Dont use '' around non-strings

**Review by Kitzunu — CHANGES_REQUESTED** [review]

**Comment by T1ti** [comment]
I deleted this fork I can't edit it anymore

**Comment by FrancescoBorzi** [comment]
we will merge this soon unless someone has anything against it.

**Pull request merged**